microsoft-edge/progressive-web-apps-chromium/landing/index.yml

```diff
@@ -0,0 +1,150 @@
+### YamlMime:Landing
+# landing page for PWA portion of docset
+title: Progressive Web Apps (PWAs) documentation
+summary: Develop a web app that has a progressive UX and can run on all devices, from one codebase.  Provide a native-like UX, or run in the browser.  # < 160 chars; cursor 169
```

Comment:
optimize 160-char desc of PWAs in pwa landing page
https://review.learn.microsoft.com/en-us/microsoft-edge/progressive-web-apps-chromium/landing/?branch=pr-en-us-3320

Reply:
```suggestion
summary: Develop a web app that can be installed and run on all devices, from one codebase. Provide a native-like UX, or run in the browser.
```